Remove Hugo.

The Hugo interpreter has a restrictive license (licenses/HUGO License.txt)
which makes it unsuitable for inclusion in this package.

There is some hope that this situation will change in the future:
http://groups.google.com/group/garglk-dev/browse_thread/thread/00f35c1cbc01f203

Index: gargoyle-fedora/src/License.txt
===================================================================
--- gargoyle-fedora.orig/src/License.txt
+++ gargoyle-fedora/src/License.txt
@@ -18,8 +18,6 @@ Git is copyright (c) Iain Merrick. It is
 
 Glulxe is copyright (c) Andrew Plotkin.
 
-Hugo is copyright (c) Kent Tessman. It is released with permission under the terms of the HUGO license.
-
 JACL is copyright (c) Stuart Allen.  It is released under the terms of the GNU General Public License.
 
 Level 9 is copyright (c) Glen Summers, David Kinder, Alan Staniforth, Simon Baldwin and Dieter Baron. It is released under the terms of the GNU General Public License.
Index: gargoyle-fedora/src/garglk/launcher.c
===================================================================
--- gargoyle-fedora.orig/src/garglk/launcher.c
+++ gargoyle-fedora/src/garglk/launcher.c
@@ -33,7 +33,6 @@
 #define T_ALAN2     "alan2"
 #define T_ALAN3     "alan3"
 #define T_GLULX     "git"
-#define T_HUGO      "hugo"
 #define T_JACL      "jacl"
 #define T_LEV9      "level9"
 #define T_MGSR      "magnetic"
@@ -341,9 +340,6 @@ int rungame(char *path, char *game)
     if (!strcasecmp(ext, "ulx"))
         return winterp(path, strcat(exe,T_GLULX), "", game);
 
-    if (!strcasecmp(ext, "hex"))
-        return winterp(path, strcat(exe,T_HUGO), "", game);
-
     if (!strcasecmp(ext, "jacl"))
         return winterp(path, strcat(exe,T_JACL), "", game);
     if (!strcasecmp(ext, "j2"))
Index: gargoyle-fedora/src/garglk/launcher.sh
===================================================================
--- gargoyle-fedora.orig/src/garglk/launcher.sh
+++ gargoyle-fedora/src/garglk/launcher.sh
@@ -58,7 +58,6 @@ case "$lowpath" in
     *.asl ) $dirpath/geas "$1" ;;
     *.cas ) $dirpath/geas "$1" ;;
     *.ulx ) $dirpath/git "$1" ;;
-    *.hex ) $dirpath/hugo "$1" ;;
     *.jacl ) $dirpath/jacl "$1" ;;
     *.j2  ) $dirpath/jacl "$1" ;;
     *.l9  ) $dirpath/level9 "$1" ;;
Index: gargoyle-fedora/src/garglk/launchgtk.c
===================================================================
--- gargoyle-fedora.orig/src/garglk/launchgtk.c
+++ gargoyle-fedora/src/garglk/launchgtk.c
@@ -45,7 +45,6 @@ char *filterlist[] =
 "AGT Games (*.agx)|*.agx;*.d[0-9][0-9]",
 "Alan Games (*.acd,*.a3c)|*.acd;*.a3c",
 "Glulx Games (*.ulx)|*.ulx;*.blb;*.blorb;*.glb;*.gblorb",
-"Hugo Games (*.hex)|*.hex",
 "JACL Games (*.jacl,*.j2)|*.jacl;*.j2",
 "Level 9 (*.l9)|*.l9;*.sna",
 "Magnetic Scrolls (*.mag)|*.mag",
@@ -57,7 +56,7 @@ char *filterlist[] =
 "All Files|*",
 };
 
-const int filtercount = 15;
+const int filtercount = 14;
 
 void winstart(void)
 {
Index: gargoyle-fedora/src/terps/Jamfile
===================================================================
--- gargoyle-fedora.orig/src/terps/Jamfile
+++ gargoyle-fedora/src/terps/Jamfile
@@ -15,7 +15,7 @@ MAKE_FROTZ      ?= yes ;
 MAKE_GEAS       ?= yes ;
 MAKE_GIT        ?= yes ;
 MAKE_GLULXE     ?= yes ;
-MAKE_HUGO       ?= yes ;
+MAKE_HUGO       ?= no  ;
 MAKE_JACL       ?= yes ;
 MAKE_LEVEL9     ?= yes ;
 MAKE_MAGNETIC   ?= yes ;
